CedarJava/src/main/java/com/cedarpolicy/model/DetailedError.java

```diff
@@ -0,0 +1,93 @@
+package com.cedarpolicy.model;
```

Comment:
Do we want to drop an explicit note that this is an encoding of `miette::Report`?

Reply:
I think that's a Rust implementation detail.  As far as the Java is concerned, this is just the structure we get over JSON from Rust.  Having that comment on the Rust side is good though